datastore/redis.go

```diff
@@ -77,10 +69,11 @@ type RedisCache struct {
 	keyKnownValidators                string
 	keyValidatorRegistrationTimestamp string
 
-	keyRelayConfig        string
-	keyStats              string
-	keyProposerDuties     string
-	keyBlockBuilderStatus string
+	keyRelayConfig       string
+	keyStats             string
+	keyProposerDuties    string
+	keyBuilderStatus     string
```

Comment:
I regret suggesting changing `BlockBuilder` to just `Builder`. (Didn't realise the existing codebase uses `BlockBuilder`.) May be worth reverting back to block builder in various places for consistency and to reduce the diff size.

Reply:
will go through again. I changed this redis key naming back to cut down the diff a bit.